pkg/arvo/app/chat-hook.hoon

```diff
@@ -0,0 +1,372 @@
+/-  *permission-store, *chat-hook
```

Comment:
Same as in the other PR, these should all have brief comments at the top.

Reply:
Good point, thanks.